File: AgendaUniversitaria/src/com/untref/au/agendauniversitaria/model/Usuario.java
package com.untref.au.agendauniversitaria.model;

import java.util.List;

public class Usuario {
	
	private String mUsuario;
	
	private String mContraseña;
	
	private String mNombre;
	
	private List <Materia> mListaMaterias;

	public String getmUsuario() {
		return mUsuario;
	}

	public void setmUsuario(String mUsuario) {
		this.mUsuario = mUsuario;
	}

	public String getmContraseña() {
		return mContraseña;
	}

	public void setmContraseña(String mContraseña) {
		this.mContraseña = mContraseña;
	}

	public String getmNombre() {
		return mNombre;
	}

	public void setmNombre(String mNombre) {
		this.mNombre = mNombre;
	}

	public List<Materia> getmListaMaterias() {
		return mListaMaterias;
	}

	public void setmListaMaterias(List<Materia> mListaMaterias) {
		this.mListaMaterias = mListaMaterias;
	}

}

File: AgendaUniverstaria-Api/ClasesAgendaUniversitaria/ClasesAgendaUniversitaria/Usuario.vb
Public Class Usuario


    Private mUsuario As String
    Public Property Usuario() As String
        Get
            Return mUsuario
        End Get
        Set(ByVal value As String)
            mUsuario = value
        End Set
    End Property


    Private mContraseña As String
    Public Property Contraseña() As String
        Get
            Return mContraseña
        End Get
        Set(ByVal value As String)
            mContraseña = value
        End Set
    End Property


    Private mNombre As String
    Public Property Nombre() As String
        Get
            Return mNombre
        End Get
        Set(ByVal value As String)
            mNombre = value
        End Set
    End Property


    Private mListaMaterias As List(Of Materia)
    Public Property ListaMaterias() As List(Of Materia)
        Get
            Return mListaMaterias
        End Get
        Set(ByVal value As List(Of Materia))
            mListaMaterias = value
        End Set
    End Property

    Public Sub agregarMateria(ByVal pMateria As Materia)
        mListaMaterias.Add(pMateria)
    End Sub

    Public Sub New()
        mListaMaterias = New List(Of Materia)
    End Sub

End Class
